pkg/monitors/ntp/ntp.go

```diff
@@ -0,0 +1,76 @@
+package ntp
+
+import (
+	"context"
+	"time"
+
+	"github.com/signalfx/golib/v3/datapoint"
+	"github.com/signalfx/signalfx-agent/pkg/core/config"
+	"github.com/signalfx/signalfx-agent/pkg/monitors"
+	"github.com/signalfx/signalfx-agent/pkg/monitors/types"
+	"github.com/signalfx/signalfx-agent/pkg/utils"
+	"github.com/sirupsen/logrus"
+
+	"github.com/beevik/ntp"
+)
+
+const minIntervalSeconds = 1800
```

Comment:
```suggestion
const minIntervalSeconds = 30 * time.Minutes
```

Reply:
It would no longer be `Seconds` at that point, but the base time.Time unit (nanoseconds), so the var should just be called `minInterval`.